{"uid":"87c1a68728e3a18e","name":"test_send_message_2_add_remove_reactions_and_cleanup[chrome]","fullName":"test_DiscordFrontEnd.TestDiscordMessageActions#test_send_message_2_add_remove_reactions_and_cleanup","historyId":"b4555d1507a1b8305e77637bd4ae3e13","time":{"start":1730123686982,"stop":1730123686982,"duration":0},"status":"broken","statusMessage":"ValueError: Email или пароль не заданы в переменных окружения.","statusTrace":"driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"f56791473d74a2f4254f4ff54124e10d\")>\n\n    @pytest.fixture(scope=\"class\", autouse=True)\n    def setup(driver):\n        login_page = LoginPage(driver)\n    \n        with allure.step(\"Открытие страницы логина\"):\n            login_page.open(\"https://discord.com/login\")\n    \n            if not login_page.is_element_visible(LoginPage.EMAIL_INPUT):\n                raise AssertionError(\"Поле для ввода email не найдено.\")\n    \n        with allure.step(\"Вход в систему\"):\n>           login_page.login()\n\nconftest.py:57: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <login_page.LoginPage object at 0x7f226c3aba30>\n\n    @allure.step(\"Вход в систему с зарегистрированными данными (аутентификация)\")\n    def login(self):\n        # Загрузка переменных окружения\n        load_dotenv('1.env')\n        email = os.getenv('DISCORD_EMAIL')\n        password = os.getenv('DISCORD_PASSWORD')\n    \n        # Проверка, что email и password не пустые\n        if not email or not password:\n>           raise ValueError(\"Email или пароль не заданы в переменных окружения.\")\nE           ValueError: Email или пароль не заданы в переменных окружения.\n\nlogin_page.py:35: ValueError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1730123682124,"stop":1730123683586,"duration":1462},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup","time":{"start":1730123683586,"stop":1730123686929,"duration":3343},"status":"broken","statusMessage":"ValueError: Email или пароль не заданы в переменных окружения.\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/DiscordFrontEndSelenium/DiscordFrontEndSelenium/conftest.py\", line 57, in setup\n    login_page.login()\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/DiscordFrontEndSelenium/DiscordFrontEndSelenium/login_page.py\", line 35, in login\n    raise ValueError(\"Email или пароль не заданы в переменных окружения.\")\n","steps":[{"name":"Открытие страницы логина","time":{"start":1730123683586,"stop":1730123686928,"duration":3342},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"Вход в систему","time":{"start":1730123686928,"stop":1730123686929,"duration":1},"status":"broken","statusMessage":"ValueError: Email или пароль не заданы в переменных окружения.\n","statusTrace":"  File \"/home/runner/work/DiscordFrontEndSelenium/DiscordFrontEndSelenium/conftest.py\", line 57, in setup\n    login_page.login()\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/DiscordFrontEndSelenium/DiscordFrontEndSelenium/login_page.py\", line 35, in login\n    raise ValueError(\"Email или пароль не заданы в переменных окружения.\")\n","steps":[{"name":"Вход в систему с зарегистрированными данными (аутентификация)","time":{"start":1730123686928,"stop":1730123686929,"duration":1},"status":"broken","statusMessage":"ValueError: Email или пароль не заданы в переменных окружения.\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/DiscordFrontEndSelenium/DiscordFrontEndSelenium/login_page.py\", line 35, in login\n    raise ValueError(\"Email или пароль не заданы в переменных окружения.\")\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":3,"attachmentsCount":0,"hasContent":true}],"afterStages":[{"name":"driver::0","time":{"start":1730123686990,"stop":1730123687056,"duration":66},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"order(2)"},{"name":"tag","value":"@pytest.mark.usefixtures('setup')"},{"name":"suite","value":"test_DiscordFrontEnd"},{"name":"subSuite","value":"TestDiscordMessageActions"},{"name":"host","value":"fv-az1272-401"},{"name":"thread","value":"2199-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_DiscordFrontEnd"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"driver","value":"'chrome'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":4,"skipped":0,"passed":0,"unknown":0,"total":4},"items":[{"uid":"d2ee95345c97ce77","reportUrl":"https://Helgi-P.github.io/DiscordFrontEndSelenium/5//#testresult/d2ee95345c97ce77","status":"broken","statusDetails":"TypeError: 'NoneType' object is not iterable","time":{"start":1730123224544,"stop":1730123224544,"duration":0}},{"uid":"66cc0c2aae49d42d","reportUrl":"https://Helgi-P.github.io/DiscordFrontEndSelenium/4//#testresult/66cc0c2aae49d42d","status":"broken","statusDetails":"TypeError: 'NoneType' object is not iterable","time":{"start":1730122254067,"stop":1730122254067,"duration":0}},{"uid":"3a82ae18d6f6e6a9","reportUrl":"https://Helgi-P.github.io/DiscordFrontEndSelenium/3//#testresult/3a82ae18d6f6e6a9","status":"broken","statusDetails":"TypeError: 'NoneType' object is not iterable","time":{"start":1730122001422,"stop":1730122001422,"duration":0}}]},"tags":["order(2)","@pytest.mark.usefixtures('setup')"]},"source":"87c1a68728e3a18e.json","parameterValues":["'chrome'"]}